action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Your name here'
+description: 'Provide a description here'
+author: 'Your name or organization here'
+inputs:
+  BRANCH_NAME:
+    required: true
+    description: "the name of the current branch"
+  ENVIRONMENT_NAME:
+    required: true
+    description: "the name of the environment variable"
+  BUCKET_NAME:
+    required: true
+    description: 'AWS S3 bucket to push the data'
+  REGION:
+    required: true
+    description: 'AWS S3 bucket region'
+  CONFIG_PATH:
+    required: true
+    description: 'Path to config file used in action'
+  TARGET_BRANCH:
+    required: true
+    description: 'Target branch of PR'
+  
+runs:
+  using: 'node16'
+  main: 'dist/index.js'
```

Comment:
Don't checkin dist files to git repo

Reply:
The compiled files are in lib directory. And we are not checkin the lib directory on github.
We will have to have the dist directory for the action to run.
Is it okay?